protocol.html

```diff
@@ -805,6 +805,12 @@ <h3 property="schema:name">Subscription Types Extension</h3>
 
                     <figcaption property="schema:name">Subscription request with a custom subscription type.</figcaption>
                   </figure>
+
+                  <p>
+                    While not necessary, we encourage authors to register subscription types with the Solid project, allowing them to be discovered by the Solid community.
+                    The process for registration shall be avaiable at the <a href="https://github.com/solid/notifications">Solid Notifications Protocol Repository</a>
```

Comment:
I don't think we should introduce terminology like register/registration - it sounds more heavy duty process than what's intended I think. Would you consider rephrasing this? What you had in the README and also what I suggested is closer to what we want.

Reply:
Does "inform" sound better? I can draft some language accordingly!

Reply:
I think the text need not be too different between the README and what's in the spec. We can potentially copy/paste.

Reply:
copy/paste sounds ok